pandas/tests/test_graphics.py

```diff
@@ -2518,12 +2512,12 @@ def test_df_legend_labels(self):
         self._check_legend_labels(ax, labels=['a', 'b (right)', 'c', 'g', 'h', 'i'])
 
         # scatter
-        ax = df.plot(kind='scatter', x='a', y='b', label='data1')
+        ax = df.plot(kind='scatter', x='a', y='b', label='data1', legend=True)
```

Comment:
When you change the default for legend back to True you can remove the `legend=True`s that you added.